### ISSUE TITLE
The value of the validation accuracy is very low 

### ISSUE THREAD

**Issue by abdou31** [tensorflow/tensorflow#33697]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10 x64
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on a mobile device:
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below):1.13.1
- Python version:3.6.4
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:9/7.5
- GPU model and memory: Nvidia Geforce 840m

I have a trained model that detects an eye region with landmarks, this model gives good results:

![image](https://user-images.githubusercontent.com/19480228/67513317-e0e4d380-f69a-11e9-88b6-aa8327d4bc58.png)

But when I evaluate this model using this code:
```python
def _eval_input_fn():
    """Function for evaluating."""
    return input_fn(
        record_file="./validiris.record",
        batch_size=2,
        num_epochs=1,
        shuffle=False)
def main(unused_argv):
    """MAIN"""
    # Create the Estimator
    estimator = tf.estimator.Estimator(
        model_fn=cnn_model_fn, model_dir="./irismodel")

    # Choose mode between Train, Evaluate and Predict
    mode_dict = {
        'train': tf.estimator.ModeKeys.TRAIN,
        'eval': tf.estimator.ModeKeys.EVAL,
        'predict': tf.estimator.ModeKeys.PREDICT
    }
    mode = mode_dict['eval']
if mode == tf.estimator.ModeKeys.TRAIN:
        estimator.train(input_fn=_train_input_fn, steps=200000)

        # Export result as SavedModel.
        estimator.export_savedmodel('./saved_model', serving_input_receiver_fn)
 elif mode == tf.estimator.ModeKeys.EVAL:
        evaluation = estimator.evaluate(input_fn=_eval_input_fn)
        print(evaluation)

```

I got an unbelievable value of accuracy, as you can see that was the logs that I have got:

**Other info / logs**
>2019-10-24 19:51:54.628843: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2019-10-24 19:53:57.017603: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-10-24 19:53:57.026100: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0
2019-10-24 19:53:57.033023: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N
2019-10-24 19:53:58.438331: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 3058 MB memory) -> physical GPU (device: 0, name: GeForce 840M, pci bus id: 0000:03:00.0, compute capability: 5.0)
WARNING:tensorflow:From C:\Users\starinfo\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\training\saver.py:1266: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
INFO:tensorflow:Restoring parameters from ./irismodel\model.ckpt-146849
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
2019-10-24 19:54:10.280447: I tensorflow/stream_executor/dso_loader.cc:152] successfully opened CUDA library cublas64_100.dll locally
INFO:tensorflow:Finished evaluation at 2019-10-24-18:00:00
INFO:tensorflow:Saving dict for global step 146849: accuracy = 1.5903983e-06, global_step = 146849, loss = 0.00016725865
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 146849: ./irismodel\model.ckpt-146849
{'accuracy': 1.5903983e-06, 'loss': 0.00016725865, 'global_step': 146849}

What is the problem here?

**Comment by jvishnuvardhan** [comment]
This is not Build/Installation or Bug/Performance issue. Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support and learn from your questions. GitHub is mainly for addressing bugs in installation and performance. Thanks!
